src/api/app/models/workflow/step/branch_package_step.rb

```diff
@@ -171,11 +152,11 @@ def create_or_update_subscriptions(branched_package, workflow_filters)
                                                               enabled: true,
                                                               token: @token,
                                                               package: branched_package)
-          subscription.update!(payload: @scm_extractor_payload.merge({ workflow_filters: workflow_filters }))
+          subscription.update!(payload: scm_extractor_payload.merge({ workflow_filters: workflow_filters }))
         end
       end
 
-      # TODO: This could be in a query object
+      # TODO: This could be in a query object. Ask Lukas, he started to work on this
```

Comment:
did we ask Lukas?

Reply:
Yes, the plan was to move it out of this model to decrease the size of it, but we simply did not manage during the last iteration since we had to work on the `report_to_scm_job`. I dont really have anything regarding this at the moment

Reply:
we can do it in another PR.. AFAIR, we need this to keep working on other cards, no?

Please: add a comment `TODO: move it to query object`, drop this comment about talking with @krauselukas and move on